examples/tutorials/streamwriter_basic_tutorial.py

```diff
@@ -0,0 +1,658 @@
+            !pip3 uninstall -y torch torchvision torchaudio
+            !pip3 install --pre torch torchvision torchaudio --extra-index-url https://download.pytorch.org/whl/nightly/cpu
+            """
+        )
+    except ModuleNotFoundError:
+        pass
+    raise
+
+print("FFmpeg library versions")
+for k, v in torchaudio.utils.ffmpeg_utils.get_versions().items():
+    print(f"  {k}: {v}")
+
+######################################################################
+#
+
+import io
+import os
+import tempfile
+
+from torchaudio.utils import download_asset
+from IPython.display import Audio, Video
+
+SAMPLE_PATH = download_asset("tutorial-assets/Lab41-SRI-VOiCES-src-sp0307-ch127535-sg0042.wav")
+WAVEFORM, SAMPLE_RATE = torchaudio.load(SAMPLE_PATH, channels_first=False)
+NUM_FRAMES, NUM_CHANNELS = WAVEFORM.shape
+
+_BASE_DIR = tempfile.TemporaryDirectory()
+
+
+def get_path(filename):
+    return os.path.join(_BASE_DIR.name, filename)
+
+
+######################################################################
+#
+# The basic usage
+# ---------------
+#
+# To save Tensor data into media formats with StreamWriter, there
+# are three necessary steps
+#
+# 1. Specify the output
+# 2. Configure streams
+# 3. Write data
+#
+# The following code illustrates how to save audio data as WAV file.
+#
+
+######################################################################
+#
+
+# 1. Define the destination. (local file in this case)
+path = get_path("test.wav")
+s = StreamWriter(path)
+
+######################################################################
+#
+
+# 2. Configure the stream. (8kHz, Stereo WAV)
+s.add_audio_stream(
+    sample_rate=SAMPLE_RATE,
+    num_channels=NUM_CHANNELS,
+)
+
+######################################################################
+#
+
+# 3. Write the data
+with s.open():
+    s.write_audio_chunk(0, WAVEFORM)
+
+######################################################################
+#
+
+Audio(path)
+
+######################################################################
+#
+# Now we look into each step in more detail.
+
+######################################################################
+#
+# Write destination
+# -----------------
+#
+# StreamWriter supports different types of write destinations
+#
+# 1. Local file
+# 2. File-like objects
+# 3. Streaming protocols (such as RTMP and UDP)
+# 4. Media devices (speakers and video players)†
+#
+# † Media devices will be explained in a separate tutorial.
+#
+
+######################################################################
+# Local files
+# ~~~~~~~~~~~
+#
+# StreamWriter supports saving media to local files.
+#
+#
+# .. code::
+#
+#    StreamWriter(dst="audio.wav")
+#
+#    StreamWriter(dst="audio.mp3")
+#
+# This works for still images and videos as well.
+#
+# .. code::
+#
+#    StreamWriter(dst="image.jpeg")
+#
+#    StreamWriter(dst="video.mpeg")
+#
+
+######################################################################
+# Streaming protocols
```

Comment:
should the order of `Streaming protocols` and `File-like objects` be reversed?

Reply:
I don't see the necessity.

Reply:
In the description the order is like
```
# 1. Local files
# 2. File-like objects
# 3. Streaming protocols (such as RTMP and UDP)
# 4. Media devices (speakers and video players)†
```
so I thought using the same order will make it consistent with the description.

Reply:
oh sorry I missed that one. Thanks, will make an update.